### PR TITLE
Add Gelma's typos from "f" to "h"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25064,11 +25087,17 @@ fiels->feels, fields, files, phials,
...
+figgure->figure
+figgured->figured
+figgures->figures
+figguring->figuring
 fightings->fighting
 figurestyle->figurestyles
 fiield->field
 fiields->fields
 filal->final
+filder->folder, filter, finder, fielder,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:25099]: Also `filer`, especially in the context of IT?

Same for plural.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -26620,6 +26732,8 @@ genric->generic
 genrics->generics
 gental->gentle
 genuin->genuine
+genuis->genius
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:26735]:
```suggestion
genuis->genius, genus,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -27160,7 +27287,8 @@ gziped->gzipped
 haa->has
 haave->have
 habaeus->habeas
-habbit->habit
+habbit->habit, rabbit,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27290]: Perhaps `hobbit` too?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -27288,6 +27417,8 @@ happend->happened, happens, happen,
 happended->happened
 happending->happening
 happends->happens
+happenend->happened
+happenes->happens, happened,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27421]: Perhaps `happiness`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -27729,14 +27869,26 @@ hopyfully->hopefully
 horicontal->horizontal
 horicontally->horizontally
 horinzontal->horizontal
+horison->horizon
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27872]: Perhaps `orison`, although rare?

Same for plural.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -27729,14 +27869,26 @@ hopyfully->hopefully
...
+horizontaly->horizontally
 horiztonal->horizontal
 horiztonally->horizontally
+horozon->horizon
+horozons->horizons
+horozontal->horizontal
+horozontally->horizontally
 horphan->orphan
 horrable->horrible
+horray->hooray
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:27891]: Can also be spelled `hurray`, but `hooray` is definitely more common:
https://books.google.com/ngrams/graph?content=hooray%2Churray&year_start=1800&year_end=2019&corpus=en-2019

Let's suggest `hooray` only.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:27872]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**